src/move_basic.cpp

```diff
@@ -259,14 +264,14 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(30.0)),
                       reverseWithoutTurningThreshold, 0.5);
 
     nh.param<std::string>("preferred_planning_frame",
-                          preferredPlanningFrame, "map");
+                          preferredPlanningFrame, "goal");
     nh.param<std::string>("alternate_planning_frame",
                           alternatePlanningFrame, "odom");
-    nh.param<std::string>("preferred_planning_frame",
+    nh.param<std::string>("preferred_driving_frame",
                           preferredDrivingFrame, "map");
     nh.param<std::string>("alternate_planning_frame",
```

Comment:
Shouldn't the param name be alternate_driving_frame? Are we just keeping this for compatibility?

Reply:
Yes, fixed.